app/models/conversations_user.rb

```diff
@@ -0,0 +1,6 @@
+class ConversationsUser < ApplicationRecord
+  belongs_to :user
+  belongs_to :conversation
+
+  validates :user, :conversation, presence: true
```

Comment:
This validation is not needed, since Rails handles them automatically for `belongs_to` associations